src/responder/pam/pamsrv_p11.c

```diff
@@ -1054,7 +1054,8 @@ static char *get_cert_prompt(TALLOC_CTX *mem_ctx,
         goto done;
     }
 
-    ret = sss_certmap_get_search_filter(ctx, der, der_size, &filter, &domains);
+    ret = sss_certmap_expand_mapping_rule(ctx, der, der_size,
+                                          &filter, &domains);
     if (ret != 0) {
         DEBUG(SSSDBG_OP_FAILURE, "sss_certmap_get_search_filter failed.\n");
```

Comment:
Debug message should be updated to reflect replacement of `sss_certmap_get_search_filter` with `sss_certmap_expand_mapping_rule`